src/test/java/redis/clients/jedis/tests/commands/AllKindOfValuesCommandsTest.java

```diff
@@ -615,6 +616,29 @@ public void scanCount() {
   }
 
   @Test
+  public void scanIsCompleteIteration() {
+    for (int i = 0; i < 100; i++) {
+      jedis.set("a" + i, "a" + i);
+    }
+
+    ScanResult<String> result = scanCompletely();
```

Comment:
WDYT if we add an assertion that `result.isCompleteIteration()` is false before the scan?. Otherwise, if `isCompleteIteration()` returns a hardcoded `true` this test wouldn't catch it.